composer.json

```diff
@@ -1,31 +1,54 @@
 {
   "name": "openconext/monitor-bundle",
   "type": "symfony-bundle",
-  "description": "A Symfony 4 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
+  "description": "A Symfony 4/5/6 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
   "keywords": ["SURFnet", "StepUp", "OpenConext", "monitoring", "health"],
   "license": "Apache-2.0",
   "minimum-stability": "stable",
   "require": {
-    "php": ">=5.4,<8.0-dev",
-    "symfony/dependency-injection": ">=3.4,<5",
-    "symfony/framework-bundle": ">=3.4,<5",
-    "webmozart/assert": "^1.2"
+    "php": ">=7.2, <9.0-dev",
+    "symfony/dependency-injection": "^4.4|^5.0|^6.0",
+    "symfony/framework-bundle": "^4.4|^5.0|^6.0",
+    "doctrine/orm": "^2.9",
+    "webmozart/assert": "^1.10"
   },
   "require-dev": {
-    "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "php-parallel-lint/php-parallel-lint": "^1.3",
     "phpmd/phpmd": "^2.6",
-    "matthiasnoback/symfony-config-test": "^2.1",
-    "phpdocumentor/reflection-docblock": "3.3.*",
-    "phpunit/php-token-stream": "1.4.*",
-    "phpunit/phpunit": "^5.7",
-    "sebastian/phpcpd": "^3.0",
-    "squizlabs/php_codesniffer": "^3.1",
-    "malukenho/docheader": "^0.1.6",
-    "mockery/mockery": "~0.9"
+    "matthiasnoback/symfony-config-test": "^4.3",
+    "phpdocumentor/reflection-docblock": "^5.2",
+    "phpunit/php-token-stream": "^3.1.3|^4.0.4",
+    "phpunit/phpunit": "^8.5|^9.0",
+    "sebastian/phpcpd": "^4.1|^5.0|^6.0",
+    "squizlabs/php_codesniffer": "^3.6",
+    "malukenho/docheader": "^0.1.8",
+    "mockery/mockery": "^1.3.5|^1.4.4"
   },
   "autoload": {
     "psr-4": {
       "OpenConext\\MonitorBundle\\": "src"
     }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "App\\Tests\\": "src/Tests/"
+    }
+  },
+  "scripts": {
+      "docheader": "vendor/bin/docheader check src/",
+      "phpcmd": "vendor/bin/phpmd src xml phpmd.xml --exclude=*/Tests/*",
+      "phpcs": "vendor/bin/phpcs src --report=full --standard=phpcs.xml --extensions=php --warning-severity=0",
+      "phpcpd": "vendor/bin/phpcpd src --exclude=src/Tests/*",
+      "phpunit": "vendor/bin/phpunit --coverage-text",
+      "post-update-cmd": [
```

Comment:
Using the `post-update-cmd` hook is a cool solution and a honest way to run these tests on Github Actions. 

But for running the tests outside of a composer install/update context, it might also be usefull to expose the set of scripts under a `tests` or other fitting script name.

Reply:
I followed you suggestion and added a **tests** section.